src/gui/qgsmapcanvas.h

```diff
@@ -523,6 +524,26 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
      */
     const QgsLabelingEngineSettings &labelingEngineSettings() const;
 
+    /**
+     * Returns true if canvas map preview jobs (low priority render jobs which render portions
+     * of the view just outside of the canvas extent, to allow preview of these
+     * out-of-canvas areas when panning or zooming out the map) are enabled
+     * for the canvas.
+     * \see setPreviewJobsEnabled()
+     * \since QGIS 3.0
+     */
+    bool previewJobsEnabled() const;
```

Comment:
Should we call this `previewEnabled()` and make it an implementation detail that we use jobs to produce them?